example.json

```diff
@@ -0,0 +1,6 @@
+{
+  "source": "./package.json",
+  "scripts": {
+    "run": "esy x example"
+  }
+}
```

Comment:
This requires the `@example` sandbox to be installed before being able to run it though. Wouldn't it be simpler to just have an `example` script in the default sandbox?

Reply:
Yes, that'd be fine! The thought here was to separate the example-usage and its dependencies and manifest completely from the actual library. But in retrospect it might be a bit of a pre-optimization. 😄